Updates for Cloud MFA and 2FA content (#7530)
magento/devdocs: src/cloud/project/user-admin.md

## Update account security settings

After you add a user to a Cloud project, ask the user to review their account settings and add the following security configuration as needed:

-  Enable two-factor authentication (TFA)

   Magento recommends adding two-factor authentication to all accounts to meet security and compliance standards. Projects configured with [MFA enforcement][mfa-enforcement-for-ssh] require two-factor authentication for all accounts that require SSH access to {{site.data.var.ece}} projects.

-  Enable SSH keys

   Users that require access to {{site.data.var.ece}} source code repositories and infrastructure must enable SSH keys on their account. See [Enable SSH keys][].

-  Create an API token

   You can generate an API token on your account that can be used for secure SSH access to an environment. You need the token to enable authentication workflows for automated processes.

>    On projects with the MFA enforcement feature enabled, you must use the API token to authenticate SSH access requests from automated accounts. to bypass authentication workflows which require two-factor authentication.

Reviewer: Removed  "feature" and period.
```suggestion
   On projects with MFA enforcement enabled, you must use the API token to authenticate SSH access requests from automated accounts to bypass authentication workflows which require two-factor authentication.
```

1. Click **Verify and save**.

>    After the account configuration update completes, Magento sends an email to the address on the account to notify about the configuration change.

Reviewer 2: "to notify them..."

>    After the account configuration update completes, Magento sends an email to the address on the account to notify about the configuration change.

Author: ```suggestion
   After the account configuration update completes, Magento sends an email to the address on the account to notify the user about the configuration change.
```

   ![Cloud TFA recovery codes]({{ site.baseurl }}/common/images/cloud/cloud-2fa-settings-tfa-recovery-codes.png){:width="550px"}

   {:.bs.callout-warning}

>    Copy the recovery codes to another location or write them down to use if you lose access to your device or authentication app. Save the codes to your account to view and manage them from your account security settings. If you lose account access on a 2FA-enabled account and have no recovery codes, you must contact your Project administrator, or submit a Magento support ticket to reset the TFA application.

Reviewer: Misc edits for clarification.
```suggestion
   Copy the recovery codes to another location or write them down in case you lose access to your device or authentication app. Save the codes to your account so you can view and manage them from your account security settings. If you lose account access on a 2FA-enabled account and have no recovery codes, you must contact your project administrator, or submit a Magento support ticket to reset the TFA application.
```
